templates/ts/sso-tab-with-obo-flow/api/package.json

```diff
@@ -2,25 +2,29 @@
   "name": "teamsfx-template-api",
   "version": "1.0.0",
   "engines": {
-    "node": "16 || 18"
+    "node": "18 || 20"
   },
+  "main": "dist/src/functions/*.js",
   "scripts": {
     "dev:teamsfx": "env-cmd --silent -f .localConfigs npm run dev",
     "dev": "func start --typescript --language-worker=\"--inspect=9229\" --port \"7071\" --cors \"*\"",
     "build": "tsc",
     "watch:teamsfx": "tsc -w",
-    "prestart": "npm run build",
+    "clean": "rimraf dist",
+    "prestart": "npm run clean && npm run build",
```

Comment:
Is the `npm run clean` necessary?

Reply:
I think prestart only be triggered when running `npm run start`, but we are running `npm run dev:teamsfx` when local debug. Is this script ever triggered?

Reply:
This code snippet is from the example code created by Azure Function Core Tools v4. I tested it without this snippet and you will upload the compiled JS file after deleting the source TS file.